app/models/kudo.rb

```diff
@@ -3,11 +3,11 @@ class Kudo < ApplicationRecord
   belongs_to :receiver, class_name: 'Employee'
   belongs_to :company_value
 
-  before_create :decrease_available_kudos
-  after_create :earned_points
+  after_create :decrease_available_kudos
```

Comment:
callbacks are considered as a code smell :) They are actually making code harder to read - you can consider refactoring it some time later on :)